src/Cabal2nix.hs

```diff
@@ -90,6 +91,7 @@ options = Options
           <*> optional (option utcTimeReader (long "hackage-snapshot" <> help "hackage snapshot time, ISO format"))
           <*> pure (\i -> Just (binding # (i, path # [ident # "pkgs", i])))
           <*> strArgument (metavar "URI")
+          <*> flag True False (long "no-submodules" <> help "don't fetch git submodules")
```

Comment:
The last nitpick: what about naming the flag
`--dont-fetch-submodules`?
Since that name is more consistent with the flag we disable.

Reply:
I personally dislike the "dont" terminology which drops the apostrophe, but `--do-not-fetch-submodules` gets even wordier.  In this case though I accede to the utility of consistency in the terminology.